policy/release/repo_ids.rego

```diff
@@ -0,0 +1,170 @@
+#
+# METADATA
+# title: Repo IDs
+# description: >-
+#   Checks that all RPMs listed in the SBOM have a valid and known
+#   repository id.
+#
+package policy.release.repo_ids
+
+import rego.v1
+
+import data.lib
+
+# METADATA
+# title: Known repo id list provided
```

Comment:
This can probably be merged with `rpm_repo_ids_valid`.

Reply:
Still open to persuasion, but I think I preferred them separate.